target/linux/ar71xx/config-4.9

```diff
@@ -22,18 +22,20 @@ CONFIG_AT803X_PHY=y
 CONFIG_ATH79=y
 # CONFIG_ATH79_DEV_AP9X_PCI is not set
 # CONFIG_ATH79_DEV_DSA is not set
-# CONFIG_ATH79_DEV_ETH is not set
-# CONFIG_ATH79_DEV_GPIO_BUTTONS is not set
-# CONFIG_ATH79_DEV_LEDS_GPIO is not set
-# CONFIG_ATH79_DEV_M25P80 is not set
-# CONFIG_ATH79_DEV_SPI is not set
-# CONFIG_ATH79_DEV_USB is not set
+CONFIG_ATH79_DEV_ETH=y
```

Comment:
This is wrong (unneeded)...
https://github.com/openwrt/openwrt/blob/master/target/linux/ar71xx/generic/config-default